style.css

```diff
@@ -0,0 +1,175 @@
+body {
+  width: 100%;
+}
+
+/* Links */
+
+body a {
+  text-decoration: none;
+}
+
+.main-footer a {
+  color: #08c;
+}
+
+.right-side-footer-container a {
+  border-right: 1px solid gray;
+  padding: 0 10px;
+}
+
+.showcase-content a {
+  color: white;
+  font-size: 25px;
+  margin-top: 35px;
+  font-family: "Lato", sans-serif;
+  font-weight: 100;
+}
+
+header .nav-bar-ul li a {
+  font: 12px/18px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+  color: #fff;
+}
+
+a.no-border {
+  border-right: #fff;
+}
+
+.showcase-content a:hover {
+  text-decoration: underline;
+}
+
+header .nav-bar-ul li a:hover {
+  text-decoration: none;
+}
+
+/* Showcase (header) Section */
+
+.header-section-container {
+  max-width: 1430px;
+  margin: 0 auto;
+}
+
+.showcase {
+  background: url(./img/main-background.jpg) no-repeat center center/cover;
+  background-position: 0 92%;
+}
+
+header {
+  height: 600px;
+}
+
+header .showcase-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  color: #fff;
+}
+
+header .nav-bar-ul {
+  display: flex;
+  width: auto;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display: flex;
}
```
**Html file**

```
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
Deleted all display: flex; properties and summarized them in one class